src/rpc/client.cpp

```diff
@@ -157,6 +157,8 @@ static const CRPCConvertParam vRPCConvertParams[] =
     { "reissuetoken", 1, "value"},
     { "burntoken", 1, "value"},
     { "transfertoken", 1, "value"},
+    { "submitpackage", 0, "rawtxs" },
+    { "submitpackage", 1, "allowhighfees" },
```

Comment:
In Bitcoin, it looks like this: 

```cpp
    { "submitpackage", 0, "package" },
    { "submitpackage", 1, "maxfeerate" },
    { "submitpackage", 2, "maxburnamount" },
```

Why are the arguments different?

Reply:
maxburnamount looks like a new argument added to bitcoin in version 25.0.(sendrawtransaction and submitpackage)
Its description is:
>Any transaction containing an unspendable output with a value greater than `maxburnamount` will
  not be submitted. At present, the outputs deemed unspendable are those with scripts that begin
  with an `OP_RETURN` code (known as 'datacarriers'), scripts that exceed the maximum script size,
  and scripts that contain invalid opcodes.
  
Do we need it?

Reply:
I see. `maxburnamount` is not needed in this PR.

About `{ "transfertoken", 1, "value"},`, let's match A to the Bitcoin notation:
https://github.com/bitcoin/bitcoin/blob/master/src/rpc/mempool.cpp#L821-L827

About `allowhighfees`, how to work?

Reply:
`allowhighfees` is like a flag  to tell Tapyrus to accept transactions even if they pay a high fee. It is used a lot in testing.